.gitignore

```diff
@@ -1,8 +1,11 @@
 src/main/java/frc/robot/GVersion.java
+src/main/java/frc/robot/subsystems/ShooterPivot*
```

Comment:
im just gonna merge this line in gitginore bc there's some weird cache with the captial 'S' ShooterPivot directory. I'll remove it on main